battery-entity.js

```diff
@@ -69,12 +69,14 @@ class BatteryEntity extends Polymer.Element {
 	}
 
 	getBatteryLevel() {
-		let batteryValue = this.stateObj.state;
-		if (this.stateObj.attributes.battery) batteryValue = this.stateObj.attributes.battery;
-		if (this.stateObj.attributes.battery_level) batteryValue = this.stateObj.attributes.battery_level;
-		return Number.isFinite(parseInt(batteryValue))
-			? parseInt(Math.round(batteryValue), 10)
-			: 0;
+		// battery value candidates in priority order
+		let candidates = [
+			this.stateObj.attributes.battery_level, 
```

Comment:
According to [this line](https://github.com/cbulock/lovelace-battery-entity/blob/9edbe110eaff86df2f07b433c766b5255fc2d285/battery-entity.js#L112) stateObj can be null.